src/ILVerify/tests/ILTests/BranchingTests.il

```diff
@@ -571,7 +571,8 @@
         lbl_ret: ret
     }
 
-    .method static public hidebysig void Branching.FromTryIntoCatch_Invalid_BranchOutOfTry_BranchIntoFilter() cil managed
+    /* These tests currently yield an assert, due to a non-null entry stack to a catch/finally block; uncomment once fixed
```

Comment:
Open issue to track the problem and link it from here?

Reply:
Opened issue: #4537